ckanext/short_urls/tests/test_plugin.py

```diff
@@ -15,6 +16,20 @@
 @pytest.mark.usefixtures('with_plugins')
 class TestPlugin(object):
 
+    def test_short_url_is_created_on_dataset_create(self):
```

Comment:
what about a test if the short_url gets translated into a valid dataset/resource url. So test that:
ckan.url/link/xxx -> ckan.url/dataset/<dataset_name>/resource/yyy
ckan.url/link/yyy -> ckan.url/dataset/<dataset_name>